<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @n33co
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Molly K. Gibson</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<!--[if lte IE 8]><script src="css/ie/html5shiv.js"></script><![endif]-->
		<script src="js/jquery.min.js"></script>
		<script src="js/jquery.scrolly.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/init.js"></script>
		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-desktop.css" />
		</noscript>
		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="css/ie/v9.css" /><![endif]-->
	</head>
	<body>

		<!-- Nav -->
		<nav id="nav">
			<ul class="container">
				<li class="active"><a href="index.html">About</a></li>
				<li><a href="GibsonCV.pdf">CV</a></li>
				<li><a href="publications.html">Publications</a></li>
				<li><a href="research.html">Research</a></li>
				<li><a href="usesthis.html">Uses This</a></li>
			</ul>
		</nav>
		<div class="wrapper style2">
			<div class="container">
			<h2>Research.</h2>
				<h3>Establishment and Dynamics of Very Low Birthweight Infant Gut Microbiota Development.					</h3>
					<p>
						Antibiotic use is nearly universal in pre-term infants, whose guts are colonized by as few as 20 bacterial species in the first weeks of life. Abnormal patterns of intestinal colonization have been associated with increased incidence of necrotizing enterocolitis (NEC) and late-onset sepsis, which combined account for over half of all mortality outside the first two weeks of life in pre-term infants. It is therefore essential to understand and quantify the impact of antibiotic treatment on the assembly and dynamics of low-diversity gut microbiota community and resistome in this population. We are studying the development and establishment of the gut microbiota and resistome in this population in Saint Louis Children's Hostpital in Saint Louis:
					</p>
					<span class="image nicu"><img src="images/NICU_07.jpg"/></span>
					<span class="image nicu"><img class="nicu" src="images/NICU_08.jpg"/></span>
					<span class="image nicu"><img class="nicu" src="images/NICU_09.jpg" /></span>
					<br /><br /><br />
					<h3>Identify fitness-enhancing factors enabling improved integration of exogenous microbes in established gut microbiota.
					</h3>
					<span class="image probiotics"><img src="images/GibsonPeseky_Fig2.png"/></span>
					<p>
						Clinical trials of probiotics have demonstrated positive outcomes when administered continuously, however after treatment is terminated, probiotics are cleared from the gut microbiota in less than three weeks in the majority of patients. The established adult microbiota excludes news species through a variety of mechanisms, and has therefore limited probiotic efficacy to short-term use and reduced their ability to treat chronic gastrointestinal diseases. We aim to engineer enhanced probiotics capable of colonizing the adult gut microbiota by using in-vivo functional metagenomics to identify genes that allow probiotic species to persist in the gut microbiota.
					</p>
					<h4>Related publications:</h4>
					<p><b>Gibson MK*</b>, Pesesky MW*, Dantas G. The Yin and Yang of Bacterial Resilience in the Human Gut Microbiota. <i>J. Mol. Biol.</i> 2014; DOI: 10.1016/j.jmb.2014.05.029
					</p>
					<h3>Improved Annotation of Antibiotic Resistance.</h3>
					<p>
						Previous work has identified antibiotic resistance genes with as low as 30% amino acid identity to any known resistance gene, demonstrating a need for identification of remote homologs, however, single coding mutations in a resistance gene can also have functional consequences. Taken together, computational annotation of antibiotic resistance is an enormous challenge and current annotation methods lack the specificity and sensitivity to accurately achieve functional annotation of this class of genes. I am interested in developing annotation methods capable of acurately classifying this extremely important class of genes. Recently, we published our antibiotic resistasnce profile hidden Markov model database, <a href="http://www.dantaslab.org/resfams/">Resfams</a>:
						<h4>Related publications:</h4>
						<p><b>Gibson MK</b>, Forsberg KJ, Dantas G. Improved annotation of antibiotic resistance determinants reveals resistomes cluster by ecology. <i>The ISME Journal</i>. 2014; 10.1038/ismej.2014.106.
						</p>
					</p>
			</div>
		</div>
	</body>
</html>